startserver.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+crontab cron_jobs
```

Comment:
can you add a comment (# Start surgery notification daemon) and maybe also (# Start flask server) before the exports below